### ISSUE TITLE
AttributeError: 'DuneClient' object has no attribute 'get_latest_result'

### ISSUE THREAD

**Issue by MSilb7** [cowprotocol/dune-client#60]
Hi, new error I started seeing today. `get_latest_result` started retuning that it doesn't exist: `AttributeError: 'DuneClient' object has no attribute 'get_latest_result'`

Not sure if it'a local environment thing or not.

Example implementation: https://github.com/ethereum-optimism/op-analytics/blob/main/contracts_tracking/test_get_result.ipynb
Query result in Dune: https://dune.com/queries/2457627

I'm on version 1.1.1

**Comment by grkhr** [comment]
Hey @MSilb7 

Please show the output of `pip show dune_client` / `pip3 show dune_client`

**Comment by MSilb7** [comment]
Yeah it must be local. I use it in a github action, and [it was fine](https://github.com/ethereum-optimism/op-analytics/actions/runs/5534607127/jobs/10100272463). Locally:

`pip show dune_client`:
```
pip show dune_client
Name: dune-client
Version: 1.1.1
Summary: A simple framework for interacting with Dune Analytics official API service.
Home-page: https://github.com/cowprotocol/dune-client
Author: Benjamin H. Smith
Author-email: ben@cow.fi
License: Apache License Version 2.0
Location: /Users/michaelsilberling/opt/anaconda3/lib/python3.9/site-packages
Requires: aiohttp, ndjson, python-dateutil, requests, types-python-dateutil, types-PyYAML, types-requests, web3
```

**Comment by MSilb7** [comment]
Yeah it's 100% a local thing. My conda environment says I have version 1.1.1, but when I print the version in my notebook, it shows 0.3.0 . I'll let you all know how I resolve it, thanks for checking!

**Comment by MSilb7** [comment]
Ok resolved it. I was dumb and upgraded dune-client in my VSCode terminal without specifically activating my conda environment to do the upgrade. So my conda environment had version 0.3.0.

I actived my conda environment, upgraded, and not it works.

Resolved, closing. Big fan of dune-client haha.
<img width="381" alt="image" src="https://github.com/cowprotocol/dune-client/assets/4006780/cf5f6ac3-1317-4aa2-8c90-72fa96139bf1">
